.gitignore

```diff
@@ -126,3 +126,6 @@ run-clang-tidy.py
 
 # ctags output
 tags
+
+# mac things
+**/.DS_Store
```

Comment:
Just `.DS_Store` is good enough, it'll ignore it in every directory in the project.

Reply:
```suggestion
.DS_Store
```